feature/experimental/bgp/ate_tests/bgp_2byte_4byte_asn/metadata.textproto

```diff
@@ -1,20 +1,19 @@
 # proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
 # proto-message: Metadata
 
-uuid: "b960ced4-2f59-49bf-b078-e53f169fac06"
-plan_id: "RT-1.19"
-description: "BGP 2-Byte and 4-Byte ASN support"
-testbed: TESTBED_DUT_ATE_2LINKS
-platform_exceptions: {
-  platform: {
-    vendor: ARISTA
+uuid:  "b960ced4-2f59-49bf-b078-e53f169fac06"
+plan_id:  "RT-1.19"
+description:  "BGP 2-Byte and 4-Byte ASN support"
+testbed:  TESTBED_DUT_ATE_2LINKS
+platform_exceptions:  {
+  platform:  {
+    vendor:  ARISTA
   }
-  deviations: {
-    route_policy_under_afi_unsupported: true
-    omit_l2_mtu: true
-    interface_enabled: true
-    default_network_instance: "default"
-    network_instance_table_deletion_required: true
+  deviations:  {
+    route_policy_under_afi_unsupported:  true
+    omit_l2_mtu:  true
+    network_instance_table_deletion_required:  true
+    interface_enabled:  true
+    default_network_instance:  "default"
   }
 }
-tags: TAGS_AGGREGATION
```

Comment:
Any reason for removing the tags in all the metadata.textproto file? Can you revert this change?

Reply:
no,

revert back it